nncf/torch/graph/graph_builder.py

```diff
@@ -134,4 +139,34 @@ def convert(dynamic_graph: DynamicGraph) -> PTNNCFGraph:
             )
 
         set_nodes_attributes_in_nncf_graph(nncf_graph)
+        # if traced_parameters:
+        #     propagate_is_shared_attribute_from_constant_nodes(nncf_graph)
         return nncf_graph
+
+
+def _propagate_true_for_is_shared_attribute(node: NNCFNode, graph: NNCFGraph):
```

Comment:
```suggestion
def _propagate_true_for_is_shared_attribute(node: NNCFNode, graph: NNCFGraph) -> None:
```

Reply:
Added, but it's not necessary, None is default for return type.